src/main/resources/locale/locale_de.properties

```
@@ -184,8 +184,8 @@ Commands.MmoInfo.Header = &3-=[]=====[]&6 MMO Info &3[]=====[]=-
 Commands.MmoInfo.Mechanics = &3-=[]=====[]&6 Funktionen &3[]=====[]=-
 Commands.MmoInfo.Mystery = &7Diese F\u00E4higkeit hast du noch nicht freigeschaltet, wenn du das tust, kannst du hier Details \u00FCber diese finden!
 Commands.MmoInfo.NoMatch = Diese F\u00E4higkeit existiert nicht!
-Commands.MmoInfo.OldSkill = &7mcMMO skills are being converted into an improved modular skill system, unfortunately this skill has not been converted yet and lacks detailed stats. The new system will allow for faster release times for new mcMMO skills and greater flexibility with existing skills.
-Commands.MmoInfo.Stats = STATS: {0}
+Commands.MmoInfo.OldSkill = &7mcMMO Skills werden in einen verbessertes modulares Skill-System konvertiert, wovon dieser Skill jedoch noch nicht betroffen ist, wodurch detaillierte Statistiken fehlen. Das neue System erm\u00F6glicht eine schnellere Ver\u00F6ffentlichung neuer mcMMO-Skills und eine gr\u00F6ßere Flexibilit\u00E4t mit bereits existenten Skills.
```

These two sub-clauses right after each other sound a bit awkward.
Since the first one is directly referencing the skill, a parenthesis might be the best solution here to convey the original meaning.
```suggestion
Commands.MmoInfo.OldSkill = &7mcMMO Skills werden in einen verbessertes und modulares Skill-System konvertiert - wovon dieser Skill jedoch noch nicht betroffen ist - weswegen detaillierte Statistiken fehlen. Das neue System erm\u00F6glicht eine schnellere Ver\u00F6ffentlichung neuer mcMMO-Skills und eine gr\u00F6ßere Flexibilit\u00E4t mit bereits existenten Skills.
```

---

I actually struggled a little bit with this sentence, somehow nothing really seems to sound completely right..

Thank you very much for your suggestions, I'll update the file once I'm done with implementing the changes :)

---

You did a good job with this though.
I am not sure if my suggestion even is the best approach here anyway.
It isn't exactly easy to translate this one.